### ISSUE TITLE
Write | Sew together

### ISSUE THREAD

**Issue by Miksel90** [Miksel90/development-platforms#18]
We need to rewrite some sections probably, and change some wording and stuff so the entire case study sound better.

**Comment by HansMariusAndreassen** [comment]
Agree. We will look at this later after we have the sections and case-study is written?

**Comment by HansMariusAndreassen** [comment]
We absolutely need to rewrite the start of some of the "chapters" as 3 start with, "Flutter, an open-source framework.."

**Comment by Miksel90** [comment]
This seems good. We just need to change the word Flutter to something else in some parts of the text.
